pkg/diff/schema_migration_plan_test.go

```diff
@@ -265,48 +271,6 @@ var (
 				},
 			},
 		},
-		{
```

Comment:
This was not testing the "how". Acceptance tests already test the same functionality but are easier to maintain. (We don't need this test case)